common/api/validata.py

```diff
@@ -16,17 +16,16 @@ def validate_file_against_schema(file, schema_url):
 
 def process_errors(report):
     errors = []
-    for error in report["tasks"][0]["errors"]:
+    for error in report["errors"]:
         errors.append(
             {
                 # cells, description, fieldNumber, name, note, rowNumber, tags, type
-                "row": error["rowPosition"],
-                "column": error["fieldPosition"],
+                "row": error["rowNumber"],
                 "field": error["fieldName"],
                 "cell": error["cell"],
                 "title": error["title"],  # Cellule vide, Format incorrect, Format de date incorrect
                 "message": error["message"],
-                "code": error["code"],
+                "code": error["type"],
```

Comment:
```suggestion
                "type": error["type"],
```
comme ca en plus on ne s'emmele pas avec les "code" HTTP

Reply:
modifié dans un commit :heavy_check_mark: